src/routes/solid-start/building-your-application/routing.mdx

```diff
@@ -77,7 +77,7 @@ export default function Index() {
 By default, the component that is rendered for a route comes from the default export of the `index.tsx` file in each folder.
 However, this can make it difficult to find the correct `index.tsx` file when searching, since there will be multiple files with that name.
 
-To avoid this, you can rename the `index.tsx` file to the name of the folder it is in and it will be rendered as the default export for that route:
+To avoid this, you can rename the `index.tsx` file to the name of the folder it is in, enclosed in parentheses. This way, it will be treated as the default export for that route:
```

Comment:
```suggestion
To avoid this, you can rename the `index.tsx` file to the name of the folder it is in, enclosed in parenthesis. 

This way, it will be treated as the default export for that route:
```